malariagen_data/anoph/dash_params.py

```diff
@@ -23,3 +23,9 @@
     int,
     "Manually override the port on which the Dash app will run.",
 ]
+
+serve_scripts_locally: TypeAlias = Annotated[
+    bool, "If True, serve scripts locally, otherwise use CDN."
+]
+
+serve_scripts_locally_default: serve_scripts_locally = True
```

Comment:
I had some problems with setting this to False, which was the previous default, so have switched back to True for now. But should test this on colab to see how well it works.